### ISSUE TITLE
.

### ISSUE THREAD

**Issue by joellemachaalany** [joellemachaalany/ift215-joelle-machaalany#1]
hello Dr.
I downloaded the git hub application on another laptop but the repository i created before isn't showing and they're asking me to clone, where should i clone the repository already created to proceed?

**Comment by eddyghabachlcu** [comment]
You can clone it on any location. But its better to clone it under: C:\Users\YOURUSERNAME\Documents\GitHub

**Comment by joellemachaalany** [comment]
well i did but i can't find the commit section

**Comment by eddyghabachlcu** [comment]
Since no modifications to be committed yet. You have to create a project in netbeans within the cloned directory. Then you commit it

**Comment by joellemachaalany** [comment]
thank you!

**Comment by joellemachaalany** [comment]
![image](https://cloud.githubusercontent.com/assets/10272035/5561740/3dea087e-8df6-11e4-9a2a-f00c24c51a13.png)

I can't use the variable names anymore. I had to create another project but it happened again.

**Comment by eddyghabachlcu** [comment]
I will check it for you

**Comment by joellemachaalany** [comment]
It's okay. I found the older folder so i replaced it and it worked. Thank you

**Comment by eddyghabachlcu** [comment]
Hope you didn't loose a lot of time on it!

**Comment by joellemachaalany** [comment]
not that much thank you

**Comment by joellemachaalany** [comment]
i'm finding difficulties in setting the limit of characters in the date of birth.

**Comment by eddyghabachlcu** [comment]
To handle the number of digits you can use a code similar to this within a keyTyped event:

if(jTextField1.getText().length()>=2){
evt.consume();
}

**Comment by joellemachaalany** [comment]
i couldn't validate only numeric entries between 1 and 31. and i found this code but it didn't work.

char ch=evt.getKeyChar();
        if (!(Character.isDigit(ch))){
            jTextField.setText(" ");  
        }
        else { evt.consume();}

**Comment by eddyghabachlcu** [comment]
char ch=evt.getKeyChar();
if (!(Character.isDigit(ch))){
evt.consume();
}

This way it will work. What you want is cancel the event if the entered char is not a digit, so if char entered in event is not a digit evt.consume.

**Comment by eddyghabachlcu** [comment]
Now for validating the entries between 1 and 31 you can do it in several ways: on  key typed, on focus lost, or even when the users presses submit (which is the easiest way: in case text field content is wrong you display a dialog message to the user).

**Comment by joellemachaalany** [comment]
it worked thank you. 
i will try it out thank you again.

**Comment by joellemachaalany** [comment]
I'm almost done with my project but i can't make the auto selection of the radio button while choosing a certain item in the combo box. I used this code, is it right??

if (cbxTitle.getSelectedItem().equals("Mr.")) {
            rbMale.isSelected();
        }

```
    else {
        rbFemale.isSelected();
    }
```

**Comment by eddyghabachlcu** [comment]
please commit and synch

**Comment by eddyghabachlcu** [comment]
to select a checkbox you must use: rbMale.setSelected(true);
rbMale.isSelected() returns a boolean.

**Comment by joellemachaalany** [comment]
i didn't get how it should be done!

**Comment by eddyghabachlcu** [comment]
if (cbxTitle.getSelectedItem().equals("Mr.")) {
rbMale.setSelected(true);
}

```
else {
    rbFemale.setSelected(false);
}
```

**Comment by joellemachaalany** [comment]
thank you!
Done, i finished it.

**Comment by eddyghabachlcu** [comment]
I will take a look and get back to you.

**Comment by eddyghabachlcu** [comment]
If you test your application you will discover many weakness points. But in normal conditions it works fine. I won't ask you for more since it's the first app, so you did a great effort, go on and enjoy new year.

**Comment by joellemachaalany** [comment]
yes I know but i couldn't fix it! it just happened suddenly.But thank you. Happy new year :smile:
